src/components/landingPage.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from "react";
```

Comment:
Let us stick to a convention for naming components. `App.js` is pascal cased while `landingPage.js` is camel cased.

Reply:
Ok noted.